src/client/__tests__/client.test.ts

```diff
@@ -0,0 +1,254 @@
+// following ts-ignore + eslint-disable fixes "Could not find declaration file for module" error for await-handler
```

Comment:
This is not really a 100% changed file. Removed array wrapper around tests which probably caused this.